### ISSUE TITLE
Plugin is incompatible with Gradle 5

### ISSUE THREAD

**Issue by grv87** [mmalohlava/gradle-visteg#12]
Gradle 5 doesn't have `org.gradle.execution.taskgraph.TaskExecutionPlan` class and maybe some others.

**Comment by bedo2991** [comment]
Does v. 1.0.3 already contain this fix? If yes, it still does not work on 5.2.1

**Comment by mmalohlava** [comment]
Published v1.0.5 into Gradle portal, however waiting for approval

```
Publishing plugin cz.malohlava.visteg version 1.0.5
Thank you. Your new plugin cz.malohlava.visteg has been submitted for approval by Gradle engineers. The request should be processed within the next 24 hours, at which point you will be contacted via email.
```

**Comment by mmalohlava** [comment]
Published here: https://plugins.gradle.org/plugin/cz.malohlava.visteg

Thank you for contribution!
